server/data/nl/danielsjf.txt

```diff
@@ -2871,3 +2871,507 @@ Onze droger is alweer stuk.
 Bij de rode lichten ga je rechtdoor.
 Geen zorgen, ik heb een vouwfiets bij me.
 Je mag niet zomaar binnen in de milieuzone.
+Ik kon er de ironie wel van inzien.
+De straaljager had nog net voldoende brandstof om  veilig op het vliegdekschip te landen. 
+De medicijnen maakten haar paranoia. 
+Hij was een echte vakman.
+Zij werkt met veel liefde voor haar vak.
+De juffrouw en de meester bereidden altijd samen hun lessen voor.
+Hij kon zijn emoties niet de baas.
+De directrice was streng maar rechtvaardig.
+Hij moest nu beslissen hoe hij verder wou met zijn leven.
+Enkel de waarheid telt.
+Zij heeft een stevige ruggengraat.
+Hij had veel haar op zijn tanden. 
+Vroeger was alles beter.
+Ik verdoen mijn tijd aan het antwoorden op zijn vragen.
+Hij huilde als een klein kind bij het zien van die film.
+De hond zocht tevergeefs naar zijn baasje.
+In de zoo hebben we een koala gezien. 
+Ze hadden haar verdooft met een paardenmiddel.
+De bokser lag knock-out op de mat.
+Haar opmerking was de genadeslag.
+Achteraf gezien was dit het kantelpunt geweest.
+Vanaf toen ging alles bergaf.
+Na enkele uren was de titanenstrijd eindelijk beslecht.
+Ze was in de wolken.
+Het cadeau was te groot om alleen te dragen.
+Het gevolg was dat we het met vier man moesten binnendragen.
+De koerier moest lopen om er nog op tijd te geraken.
+Tot driemaal toe had hij aangebeld. 
+Haar repliek kwam als een donderslag bij heldere hemel.
```

Comment:
repliek => antwoord (BE/NL)

Reply:
Wordt repliek niet gebruikt in NL? Het zijn ook niet altijd de woorden die ik het meest gebruik, maar ik probeer er ook de wat minder bekende woorden er ook in te verwerken :-)

https://www.encyclo.nl/begrip/repliek